bluebrain/repo-bluebrain/packages/py-bglibpy/package.py

```diff
@@ -24,6 +24,7 @@ class PyBglibpy(PythonPackage):
     depends_on('py-cachetools', type='run')
     depends_on('py-bluepy@2.4.2:2.999', type='run')
     depends_on('py-bluepy-configfile@0.1.18:', type='run')
+    depends_on('py-pandas@1.4.0:', type='run')
```

Comment:
We don't have pandas 1.4.0 in Spack yet, that's why stuff is failing :)

Reply:
Thanks! It's updated now :)